src/main/java/it/gov/pagopa/pu/debtpositions/repository/DebtPositionTypeOrgOperatorsRepository.java

```diff
@@ -7,4 +7,5 @@
 @RepositoryRestResource(path = "debt-position-type-org-operators")
 public interface DebtPositionTypeOrgOperatorsRepository extends JpaRepository<DebtPositionTypeOrgOperators,Long>{
 
+  DebtPositionTypeOrgOperators findDebtPositionTypeOrgOperatorsByOperatorExternalUserId(String operatorExternalUserId);
```

Comment:
Entity name is not required
```suggestion
  DebtPositionTypeOrgOperators findByOperatorExternalUserId(String operatorExternalUserId);
```